### PR TITLE
Minor dashboards fixes, grafana update related changes

### PR DESCRIPTION
**Pull request by nflaig**
**Motivation**

As suggested in https://github.com/ChainSafe/lodestar/pull/5200#pullrequestreview-1314823368, the initial PR https://github.com/ChainSafe/lodestar/pull/5200 should be split into two separate PRs

**Description**

- Almost all the changes in this PR are related to a newer grafana version
- Also fixes broken panel in vm+host dashboard which could not be edited, I think this was related to incorrect x/y coordinates
- Typo fix and few title updates, nothing special..

### CODE REVIEW

File: dashboards/lodestar_block_processor.json [hunk 1]
```diff
@@ -3537,7 +3537,7 @@
       "collapsed": false,
       "datasource": {
         "type": "prometheus",
-        "uid": "prometheus_local"
+        "uid": "${DS_PROMETHEUS}"
```
Comment (dapplion) [inline dashboards/lodestar_block_processor.json:3540]: This was not caught by the dashboard linter? Could you take a look @nflaig

**Review by dapplion — APPROVED** [review]

File: dashboards/lodestar_block_processor.json (hunk 1 above)
Comment (nflaig) [reply dashboards/lodestar_block_processor.json:3540]: yeah something is off with the linter, I will take a look at that.

Comment (nflaig) [reply dashboards/lodestar_block_processor.json:3540]: Fixed in https://github.com/ChainSafe/lodestar/pull/5221

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.6.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.6.0) :tada:

**Pull request merged**